guardian/pan-domain-authentication
# Migrate CI from Teamcity to Github Actions

## What does this change?
This migrates CI from TeamCity to GitHub Actions.

## How to test
Check the build succeeds in GH Actions. This project is made up of libraries which are released via the command line, so there is no build in riffraff. The action is used to build and test changes to make sure no issues have been introduced.

## Rollout

- [x]  Pause the TeamCity build
- [ ]  Adjust branch protection rules, requiring the GHA build to pass before merge
- [ ]  Ensure all open PRs are rebased against main to ensure their build runs
